go.mod

```diff
@@ -5,94 +5,111 @@ go 1.23
 toolchain go1.23.1
 
 require (
-	github.com/bufbuild/buf v1.30.0
+	github.com/bufbuild/buf v1.41.0
 	github.com/crossplane/crossplane-runtime v1.17.0
 	github.com/go-json-experiment/json v0.0.0-20240815175050-ebd3a8989ca1
 	github.com/go-logr/logr v1.4.2
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/go-cmp v0.6.0
 	github.com/pkg/errors v0.9.1
-	github.com/upbound/provider-aws v0.47.4
+	github.com/upbound/provider-aws v1.13.1
 	go.uber.org/zap v1.27.0
 	google.golang.org/grpc v1.66.2
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.5.1
-	google.golang.org/protobuf v1.34.2
+	google.golang.org/protobuf v1.34.3-0.20240816073751-94ecbc261689
```

Comment:
Any particular reason not to bump it just to 1.34.3?

Reply:
Not sure - this is just what the Go dependency solver decided on. 🤷

Reply:
Apparently it doesn't exist yet:

```
go: google.golang.org/protobuf@v1.34.3: invalid version: unknown revision v1.34.3
```